Propose Quick start (#6)
opengisch/qgis_geomapfish_locator: README.md

# Geomapfish Locator Plugin
A QGIS 3 locator plugin for Geomapfish services

## Quick start Guide

> Go to Extension -> Install and search for Geomapfish Locator

Reviewer: ```suggestion
Go to `Plugins` -> `Manage and Install Plugins` and search for Geomapfish Locator
```

Install the plugin

> Go to the locator options by clicking on the magnifier on the bottom left of the screen

Reviewer: ```suggestion
`Settings` -> `Options` -> `Locator` and open the configuration dialog for `Geomapfish Locator`
```

> Set filter parameters for your search service :

Reviewer: ```suggestion
Set the parameters of your service :
```

* Name of the filter (e.g. MysearchService)

> * URL (e.g. https://www.sigip.ch/search)

Reviewer: ```suggestion
* URL (e.g. https://www.myservice.ch/search)
```

* CRS (e.g. EPSG 21781)

> Click OK and search

Reviewer: ```suggestion
Validate and start search in the locator bar at the bottom left of the application.

If you want to avoid typing the prefix of the filter (`gmf`), enable it by default in the locator bar settings.
```
